### ISSUE TITLE
django.conf.urls.patterns() is deprecated and will be removed in Django 1.10

### ISSUE THREAD

**Issue by ad-m** [django-debug-toolbar/django-debug-toolbar#807]
Hello,

I got warning:
```
site-packages/guardian/admin.py:118: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='%s_%s_permissions_manage_group' % info),
```

I suggest drop django.conf.urls.patterns for django 1.10 compatibility.

See https://github.com/django/django/blob/stable/1.9.x/django/conf/urls/__init__.py#L80-L85 and django-guardian/django-guardian#395 .

Greetings,

**Comment by aaugustin** [comment]
Yes, we should drop support for Django 1.7 now.

**Comment by aaugustin** [comment]
We did this some time ago in bb21defad12ee0ab0654d87b0654d484f46050bf.

The old style was still used in a doc page, I fixed that in c2658d13df4f8479ef356597680a62716c60ee61.
